smartcontractkit/external-adapters-js
# Fix/Readme generator issue

## Closes SC-54826

## Description
Fixing readme generator action in consume changeset not committing changes

## Changes

- Configured credentials for the GH bot needed to perform commits. Following the example of changesets/action@v1 used to create changeset PR

## Steps to Test

1. Run workflows in the auto-generated consume changeset PR to trigger readme generator and ensure READMEs are getting updated

## Quality Assurance
